sound/soc/sof/ipc.c

```diff
@@ -282,7 +282,7 @@ static int tx_wait_done(struct snd_sof_ipc *ipc, struct snd_sof_ipc_msg *msg,
 }
 
 /* send IPC message from host to DSP */
-static int sof_ipc_tx_message_unlocked(struct snd_sof_ipc *ipc, u32 header,
+static int sof_ipc_tx_message_unlocked(struct snd_sof_ipc *ipc,
```

Comment:
ah, so this answers to my previous feedback, maybe this removal can be done earlier, or announced earlier in the series.

Reply:
OK, I'll squash it to the patch which 'un-uses' the parameter.